docs/src/reconciling-local-changes.md

```diff
@@ -0,0 +1,82 @@
+# Reconciling changes made outside of TPA
+
+In general, if a cluster was created using TPA it is advisable to manage
+configuration changes to that cluster using TPA. This page is designed
+to help users understand how TPA manages configuration, what the
+implications of making local changes to a TPA-managed cluster are, and
+how to reconcile such changes.
+
+## What constitutes a configuration change?
+*This definition seems questionable with changes which wouldn't normally
+appear in the config at all, like inserting some data into a table*
+Consider a cluster, 'A'. Now imagine a change is applied to it and call
```

Comment:
Struggling to find a nice definition that is even 90% right here. It's a hard thing to generalize.